src/pattern-library/components/patterns/data/Scroll.js

```diff
@@ -0,0 +1,187 @@
+import { Scroll, ScrollContainer, ScrollContent } from '../../../../next';
+import Library from '../../Library';
+import Next from '../../LibraryNext';
+
+import { SampleListElements } from '../samples';
+
+export default function ScrollPage() {
+  return (
+    <Library.Page
+      title="Scroll components"
+      intro={
+        <>
+          <p>
+            The <code>Scroll</code> family of presentational components provide
+            support for contained, scrollable content:
+          </p>
+          <ul>
+            <li>
+              <code>ScrollContainer</code>: Contains its content — a mix of
+              scrollable and (optionally) non-scrollable content — to the
+              dimensions of a parent element
+            </li>
+            <li>
+              <code>Scroll</code>: A layout container that scrolls on overflow,
+              with shadows for scroll-hinting
+            </li>
+            <li>
+              <code>ScrollContent</code>: Applies consistent spacing and padding
+              to content in a <code>Scroll</code>
+            </li>
+          </ul>
+        </>
+      }
+    >
+      <Library.Pattern title="Status">
+        <p>
+          <code>Scroll</code>, <code>ScrollContainer</code>, and{' '}
+          <code>ScrollContent</code> are new components modeled after aspects of
+          the legacy <code>Scrollbox</code> component.
```

Comment:
Out of interest, can you expand on the rationale for the UI / API design changes from the old `Scrollbox` component to the three new ones?

Reply:
I touched on this in another comment, but the pattern I'm aiming for is composition of presentational components within a composite component, versus a single, and sometimes magical or inflexible, composite component. This gives the author the opportunity to compose things differently if there is a need for flexibility. Compare this to the current situation, in which occasionally it's like, "oh, crap, I need there to be one tick less padding on an element two levels deep in this composite component" or "the font size of a subheading absolutely must be `14px` in this context", which, believe me, comes up more often than you'd think (especially with our existing applied design in the client; it's like the English language, more inconsistencies than consistencies :) ). 

A theme I'm hearing is that we should be more clear about putting the components that are in the "standard/default" set more in the forefront, and backgrounding the lower-level presentational components that make up these default composite components a bit, as, in a worst case, they could sort of be implementation details of the composite components. This feels more like a documentation issue than anything.